pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -164,7 +164,7 @@ public void resetClusters() throws Exception {
         pulsar.getConfiguration().setForceDeleteNamespaceAllowed(true);
         for (String tenant : admin.tenants().getTenants()) {
             for (String namespace : admin.namespaces().getNamespaces(tenant)) {
-                deleteNamespaceGraceFullyByMultiPulsars(namespace, true, admin, pulsar,
+                deleteNamespaceGraceFully(namespace, true, admin, pulsar,
```

Comment:
I have a question, can we use the `admin.namespaces().deleteNamespace()` instead of this?

`deleteNamespaceGraceFully` looks like a hack operation.

Reply:
When `admin.namespaces().deleteNamespace()` and `auto create topic __change_event` are concurrent executed, there is a problem https://github.com/apache/pulsar/pull/17070, and `deleteNamespaceGraceFully` is used to solve this problem

Reply:
> I try to fix the flaky test this way: make delete namespace after __change_events is successfully created, but there still has another race condition: the `__change_events/__compaction` async creates and `__change_events` delete by namespace delete. Therefore, I will disabled systemTopic in method `testDeleteTenant` to solve this flaky test.

Do you point?

If right, I suggest we should fix this first.

Reply:
In user use, the probability of occurrence is almost 0. Even if it actually occurs, the user can try to execute the `delete namespace` again. But to solve this problem fundamentally requires a big change